internal/resolver/resolver.go

```diff
@@ -578,13 +601,43 @@ func (r *resolver) loadAsFileOrDirectory(path string) (string, bool) {
 	return "", false
 }
 
+func resolvePathWithoutStar(from, path string) (string, error) {
+	replaced := strings.Replace(path, "/*", "", -1)
+	return fp.Join(from, replaced), nil
```

Comment:
The code in this file is careful to always use the `fs` interface in the `resolver` instead of `path/filepath`. All platform-dependent things are supposed to happen inside the `fs` interface so that the resolver can be platform-independent. That interface uses the real `path/filepath` module for the esbuild command and the `path` module for tests. That way tests don't do different things on Windows.

Reply:
thank you! resolved in 444bfae